markdown_stoplight/use-cases.md

```diff
@@ -1,128 +1,157 @@
+|`GET` |[/products](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/products-list)
+|`GET`| [/categories](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/categories/categories-list)
+|`POST` |[/carts](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/carts/cart-create)
+|`PUT` |[/carts/{cartId}/line-items/{lineItemId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/carts/cart-replace-line-item)
+|`POST` |[/carts/{cartId}/order](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/carts/cart-order)
+|`GET`|[/legal-content](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/legal-content/merchant-legal-content-list) 
 
 ### Shipping
 
-Enable merchants to print shipping labels, automate parts of their shipping process, and track the status of orders, or implement notifications. Aggregate address and weight information to generate ready-made shipping labels or help a merchant spot issues with orders that have been pending for too long. This way, you can facilitate the shipping of goods for a merchant.</br>
+Enable merchants to print shipping labels, automate parts of their shipping process, and track the status of orders, or implement notifications.
+Aggregate address and weight information to generate ready-made shipping labels or help a merchant spot issues with orders that have been pending for too long.
+This way, you can facilitate the shipping of goods for a merchant.</br>
 
 A few of the endpoints to achieve this:
 
 | HTTP Verb | Endpoint
 |---|---
-|`GET`|[/orders](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list)
-|`GET`|[/products/{productId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
-|`GET`|[/order/{orderId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
-|`GET`|[/orders/{orderId}/processes/shippings](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-processes) |`GET`|[/orders/{orderId}/processes/shippings/{shippingProcessId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-process)
+|`GET`|[/orders](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list)
+|`GET`|[/products/{productId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
+|`GET`|[/order/{orderId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
+|`GET`|[/orders/{orderId}/processes/shippings](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-processes) |`GET`|[/orders/{orderId}/processes/shippings/{shippingProcessId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-process)
```

Comment:
Not sure how important this is but in the enpoint itself the id are written like this: /orders/{order-id}/processes/shippings/{shipping-process-id}

Reply:
Know issue. Has been reported to the team already.